src/status_im/contexts/chat/messenger/messages/pin/events.cljs

```diff
@@ -132,7 +132,4 @@
    cofx
    {:padding-bottom-override 21
     :content                 (fn [] [pinned-messages-menu/view
-                                     {:chat-id                     chat-id
-                                      :disable-message-long-press? (not= :chat
-                                                                         (get-in cofx
-                                                                                 [:db :view-id]))}])}))
```

Comment:
Problem: We no longer have view-id :chat, it was migrated to :screen/chat - [more details](https://github.com/status-im/status-mobile/issues/22497#issuecomment-2826974935)